docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -637,13 +664,13 @@ test('Check Label HTML', async t => {
 });
 ```
 
-> Note that selector's property getters and client functions are asynchronous. If you need their resulting value in your code, use the `await` keyword.
+> Selector's property getters and client functions are asynchronous. If you need their resulting value in your code, use the `await` keyword.
```

Comment:
`Selector's property getters and client functions are asynchronous.`
What is a property "getter"? Is this a widely-used technical term?

`If you need their resulting value in your code, use the `await` keyword.`
Is a single value or multiple values?

Reply:
1. Yes, getter is a widely-used programming term. It describes a function that is used to obtain a property

2. Each getter and each client function return one value. However, a selector can have multiple getters.

Does this work better:

> Client functions and selector's property getters are asynchronous. If you need their resulting values in your code, use the `await` keyword.

Reply:
I see. So, are we adding the resulting values to our code? For example:

`Client functions and selector property getters are asynchronous. If you need to add their resulting values to your code, use the await keyword.`

Or maybe "include"?

`Client functions and selector property getters are asynchronous. If you need to include their resulting values in your code, use the await keyword.`

Reply:
We use the value rather than add or include it.

I'd say:
> Client functions and selector property getters are asynchronous. If you need to utilize their resulting values in your code, use the await keyword.

Reply:
```suggestion
> Client functions and selector property getters are asynchronous. If you want to use their resulting values in your code, use the `await` keyword.
```